### ISSUE TITLE
[Visual requirement – Make Code – Explorer] Luminosity ratio is less than 3:1 for 'Add & Delete' icons present under 'Explorer' button.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3640]
**User Experience:**
Visual impaired user will find difficulty to see the non text control if that control or element have insufficient color contrast ratio as user won't be able to proceed further.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Java Script" drop down and select it.
6. Navigate to "Python" option and select it.
7. Navigate to "Explorer" and select it.
8. Navigate through all the controls after expanding Explorer control.
9. Verify contrast ratio of 'Add & Delete' icons by CCA tool.

**Actual Result:**
Luminosity ratio is less than 3:1 for 'Add & Delete' icons present under 'Explorer' button and due to this, visual impaired user is unable to see that icon.

**Expected Result:**
Luminosity ratio should be more than and equal to 3:1 for 'Add & Delete' icons present under 'Explorer' button, so that visual impaired user could easily such icon.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 1.4.11 - Non-text Contrast
](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.4.11, #Win10-Edge(Chromium)

![WCAG1 4 11_Explorer_CCA for Add and Delete icons](https://user-images.githubusercontent.com/72871089/98652837-3baea300-2362-11eb-80d2-c5075048956c.jpg)
